.cookiecutter/includes/HACKING.md

```diff
@@ -0,0 +1,40 @@
+
+Testing Manually
+----------------
+
+To test it manually you can install your local development copy of
+`tox-recreate` into the local development environment of another tox-using
+project such as
+[cookiecutter-pypackage-test](https://github.com/hypothesis/cookiecutter-pypackage-test):
```

Comment:
Could the cookie cutter use the repo's name on ehese URLs, path.

I looks like it does in some but not all.